test/fixtures/signs.yml

```diff
@@ -19,3 +19,9 @@ two:
   emergency: MyString
   emergency_detail: MyString
   visibility: 1
+
+private:
+  name: A Private Sign
+  template: default
+  location: Build Team Area
+  visibility: 1               # Yes, 1 means hidden/private.
```

Comment:
The visibility enum is kind of confusing. Setting visibility: 1, makes it seem like 'yes its visible'.